chicken-sloths/bangazon-cli-sprint2
# Joe add payment type ctrl

# Description
I made the controller for the main menu option of adding a payment option for a user. THIS IS A ROUGH DRAFT PR. I have not proofread it, or removed console logs. I only THINK it works. I put this up so yall could test it, and possibly roast me for it 

## Related Ticket(s)
Fixes #3 

## Problem to Solve
The user needs to be able to add a payment option to a customer.

## Proposed Changes
I realize I need to remove the console logs. using the console logs, and the database itself, you can look and see that the new row was added to the Payment_Options Table.

## Expected Behavior
It should correctly add the new payment option to the Payment_Options Table

## Steps to Test Solution

1. run `npm start`
1a. select an active customer!!! (i almost forgot to add this in)
2. select menu option `3`
3. You should see a list of the payment types. This is being pulled straight from the ARRAY of payment types in the faker file lmao. 
4. when you select one of those with a number, it will add the string payment type name to the new paymentOption object.
5. Then add an integer account number. 
6. The new payment option will be added to the database INCLUDING the active customer id. it will let you add a new payment option with no active customer selected. I need to put in a test for that. 
7. check the database to see the new payment option

## Testing


## Documentation
